sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/AesCbcHmacSha2.java

```diff
@@ -108,6 +176,74 @@
             hmac.update(iv);
         }
 
+        private static byte[] toBigEndian(long i) {
```

Comment:
Consider de-duplicating the `toBigEndian` and `getAlgorithmParameters` code in Encryptor and Decryptor classes. If we need to fix something, it will require changing at multiple places.

Reply:
Will refactor in a follow up PR, today.